charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -466,4 +466,5 @@ useOldCSIDriver: false
 
 # Supply a custom image to the ebs-csi-driver-test pod in helm-tester.yaml
```

Comment:
Could you make sure this comment sticks to `helmTester.image`?

Reply:
Hello @AndrewSirenko  
Sorry but I'm not understanding your comment. Do you want me to add a description regarding the enable flag?